infoschema/metric_table_def.go

```diff
@@ -1384,11 +1384,6 @@ var MetricTableMap = map[string]MetricTableDef{
 		PromQL: `sum(rate(tikv_coprocessor_request_duration_seconds_count{$LABEL_CONDITIONS}[$RANGE_DURATION])) by (req,instance)`,
 		Labels: []string{"instance", "req"},
 	},
-
-	"tikv_cop_total_request_errors": {
```

Comment:
Why remove this? It is different from `tikv_coprocessor_is_busy`.

Reply:
They have the same expression.

Reply:
reverted